docs/docs/recipes/protect-your-api/README.md

```diff
@@ -50,15 +50,15 @@ See [API Resource Logto Schema](../../references/resources/README.md#logto-api-r
 All the API Resources record registered in Logto **Admin Console** will be shared across all your applications.
 :::
 
-## Register the API resources to the Logto SDK
+## Register the API resources to your client application
```

Comment:
It's not like a registration process. more like integration or "add the API resource to your auth request"

Reply:
updated